src/server/api/controllers/user.controller.js

```diff
@@ -321,6 +321,16 @@ const createAdminBySuperUser = async (id, body) => {
   });
 };
 
+const getUsersbyOrganizationId = async organization_id => {
+  const usersByOrganizationId = await knex
+    .from("users")
+    .where({ organization_id });
+  if (usersByOrganizationId.length === 0) {
+    return `This Organization have no users!`;
```

Comment:
Throw an HTTP error here instead of just returning a string! :heart:

Reply:
Done ;)